rocky/reports/views/base.py

```diff
@@ -308,6 +327,13 @@ class OOISelectionView(BaseReportView, BaseOOIListView):
     Shows a list of OOIs to select from and handles OOIs selection requests.
     """
 
+    def setup(self, request, *args, **kwargs):
+        super().setup(request, *args, **kwargs)
+        object_selection = request.GET.get("object_selection", "")
```

Comment:
`object_selection` is referenced both as a query parameter as well as a body parameter, is that intentional?

Reply:
Yes, this is intentional. One time we use it in a get request (after refreshing the page) and another time we use it in a post request (when going to the next pages).